src/group_theory/subgroup/basic.lean

```diff
@@ -1005,6 +1005,19 @@ subsingleton.elim _ _
 @[simp, to_additive] lemma subgroup_of_self : H.subgroup_of H = ⊤ :=
 top_le_iff.mp (λ g hg, g.2)
 
+lemma subgroup_of_mono_left {H K : subgroup G} (L : subgroup G) (h : H ≤ K) :
```

Comment:
These lemmas are no longer used in this PR, so you might consider splitting them off into a separate PR.

Reply:
Done